modules/core/02-client/keeper/keeper_test.go

```diff
@@ -177,11 +169,6 @@ func (suite *KeeperTestSuite) TestValidateSelfClient() {
 			ibctmtypes.NewClientState(suite.chainA.ChainID, ibctmtypes.DefaultTrustLevel, trustingPeriod, ubdPeriod, maxClockDrift, testClientHeight, commitmenttypes.GetSDKSpecs(), nil, false, false),
 			true,
 		},
-		{
-			"invalid client type",
-			localhosttypes.NewClientState(suite.chainA.ChainID, testClientHeight),
-			false,
-		},
```

Comment:
refactor test case

Reply:
I thought the point here was that localhost is the invalid client type?

Reply:
yea but so is the solo machine. `ValidateSelfClient` must be called with a tendermint client

Reply:
OK yeah makes sense.